barretenberg/cpp/src/barretenberg/relations/generated/avm/gas.hpp

```diff
@@ -41,23 +41,23 @@ template <typename FF_> class gasImpl {
         {
             Avm_DECLARE_VIEWS(0);
 
-            auto tmp = (gas_sel_gas_cost - gas_sel_gas_cost);
+            auto tmp = ((gas_sel_gas_cost - gas_sel_gas_cost) - FF(0));
```

Comment:
These - FF(0) changes are unrelated, and seem to have resulted from a change in our bb-pilcom.

They're addressed in a separate PR